tests/Nelmio/Alice/ReferenceRangeNameTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *
+ * (c) Nelmio <hello@nelm.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nelmio\Alice;
+
+class ReferenceRangeNameTest extends \PHPUnit_Framework_TestCase
+{
+    const USER = 'Nelmio\Alice\support\models\User';
+    const GROUP = 'Nelmio\Alice\support\models\Group';
+    const CONTACT = 'Nelmio\Alice\support\models\Contact';
+
+    /**
+     * @test
```

Comment:
let's keep consistent with the rest of the project and prefix the test methods with `test` instead of using the `@test` annotation